src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,39 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
```

Comment:
True you can't tell by looking at the signature, but the signature is a strange place to look to figure out the meaning of the return value. If it was `std::pair<bool, std::uint32_t>` would that tell you much of anything?